README.md

```diff
@@ -1,21 +1,52 @@
-# ``cleanlab`` Examples
+# cleanlab Examples
 
-Not sure where to start? Try checking out how to find [ImageNet Label Errors](imagenet/imagenet_train_label_errors.ipynb).
+This repo contains code examples that demonstrate how to use [cleanlab](https://github.com/cleanlab) and how [confident learning](https://arxiv.org/abs/1911.00068) works to find label errors.
```

Comment:
Add sentence:

To quickly learn the basics of running cleanlab  on your own data, we recommend first starting [here](https://docs.cleanlab.ai/v1.0.1/index.html#quickstart) before diving into the examples below.

Reply:
TODO: that link I posted is intended to be for currently-unreleased  2.0.0 docs, verify it actually works once 2.0.0 is tagged

Reply:
Sounds good, I added the sentence and will verify the link works once 2.0.0 is tagged.

Reply:
@jwmueller I updated the link here to use the v2.0.0 docs and verified it.

685d5c1a6b35154659e807f9b356c8570388e57f